src/components/FeaturedEventCard.tsx

```diff
@@ -5,85 +5,99 @@ import Location from '@assets/new/icons/Location';
 import People from '@assets/new/icons/People';
 import Time from '@assets/new/icons/Time';
 import Whatsapp from '@assets/new/icons/Whatsapp';
-import { EventDataType, EventFormat } from '@data/eventsData';
-import Link from 'next/link';
+import { EventFormat } from '@data/eventsData';
+import { TrainingSupbaseDataType } from '@data/types/trainingTypes';
+import { calculateDateDifference, formatDate } from '@utils/index';
+import Image from 'next/image';
 import React, { useState } from 'react';
-import { usePaystackPayment } from 'react-paystack';
+// import { usePaystackPayment } from 'react-paystack';
 import styled from 'styled-components';
-import CTA, { CTALink } from './CTA';
+import { CTALink } from './CTA';
 import { priceFormatter } from './ProductCard';
+import FeaturedEventDialog from './FeaturedEventDialog';
 
 interface FeaturedEventProp {
-	event: EventDataType;
 	userEmail: string;
+	training: TrainingSupbaseDataType;
 }
 
-const onSuccess = (reference: any) => {
-	console.log(reference);
-};
+// const onSuccess = (reference: any) => {
+// 	console.log(reference);
+// };
 
-const onClose = () => {
-	console.log('closed');
-};
+// const onClose = () => {
+// 	console.log('closed');
+// };
 
 const FeaturedEventCard: React.FC<FeaturedEventProp> = ({
-	event,
 	userEmail,
+	training,
 }) => {
-	const config = {
-		reference: new Date().getTime().toString(),
-		email: userEmail,
-		amount: event.price * 100,
-		publicKey: process.env.NEXT_PUBLIC_PAYSTACK_KEY as string,
-	};
+	// const config = {
```

Comment:
this too, you don't need to keep them as backups when you make changes that's why we use git to track changes.